tests/unit/helpers/customerNotes.test.js

```diff
@@ -133,15 +138,16 @@ describe('update', () => {
         action: NOTE_UPDATE,
       }
     );
+    sinon.assert.callCount(customerNoteHistoryCreate, 2);
     sinon.assert.calledOnceWithExactly(
       updateOne,
       { _id: customerNote._id, company: credentials.company._id },
-      { $set: { title: 'titre', description: 'description mise a jour' } }
+      { $set: { title: 'titre mis a jour', description: 'description mise a jour' } }
     );
   });
 
   it(
-    'should not update customer note and not create history if description in payload is the same as before',
+    'should not update customer note and not create history if description and title in payload are the same as before',
```

Comment:
Que penses-tu de ce nom pour le test `should not update note or create history if neither description nor title are changed`. Ca permet de tout faire tenir sur une ligne